shared/react-native/android/app/src/main/java/io/keybase/ossifrage/MainActivity.java

```diff
@@ -38,7 +38,7 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         logFile = this.getFileStreamPath("android.log");
```

Comment:
I'm having trouble understanding what's going on here with `logFile` after this change. It's used in `onKeyUp` below, but does that actually do something?